### PR TITLE
doc: Add two entries to the FaQ about channel management

### PR DESCRIPTION
**Pull request by cdecker**
Changelog-None

### CODE REVIEW

**Comment by niftynei** [comment]
the spellchecker caught a `lighting` bug :wink: 

```
```

File: doc/FAQ.md [hunk 1]
```diff
@@ -131,9 +131,109 @@ Tools for replication are currently in active development, using the `db_write`
...
+```
+
+This will store a unilateral close TX in the DB as last resort means
+of recovery should the channel unexpectedly confirm anyway.
+
+Now you can use the `dev-forget-channel` command to remove
+the DB entries from the database.
+
+```bash
+lightning-cli dev-forget-channel $NODEID
```
Comment (niftynei) [inline doc/FAQ.md:179]: Needs a note that to access the `dev-` commands you need to be using a developer build.

Comment (cdecker) [reply doc/FAQ.md:179]: Added a quick notice about the `dev-` commands only being available in `DEVELOPER=1` builds

**Comment by cdecker** [comment]
> the spellchecker caught a `lighting` bug wink

Indeed, muscle memory betrays me evey once in a while 😆

**Comment by niftynei** [comment]
ACK https://github.com/ElementsProject/lightning/pull/5394/commits/ec59b8784a2bbc6138d5de097860bbaf0622d085

test fails are all flakes

**Pull request merged**